### ISSUE TITLE
resource are not generating properly

### ISSUE THREAD

**Issue by sunildhiman90** [icerockdev/moko-resources#511]
Hi, in my project, resources are not generating.
Not able to use strings with MR.strings
Similarily for all other resources, it show MR.directory_name, but not any resource is accesible
And for strings, it does not show even strings with MR, like this: MR.strings, not available

Reproducer:
You can checkout given repo, and select **with-moko-resources-library** branch
https://github.com/sunildhiman90/compose-multiplatform-template-with-navigation

I have attached screenshot as well for project resources structure with moko resources.
<img width="445" alt="Screenshot 2023-06-06 at 10 31 38 AM" src="https://github.com/icerockdev/moko-resources/assets/35147222/018fde5d-f34e-4a27-b526-e304f98cdeb0">

Thanks.

**Comment by Alex009** [comment]
hi!
did you call `generateMRcommonMain` task after changes of resources directory? 
i clone your repo and all works fine as i see.
<img width="1512" alt="image" src="https://github.com/icerockdev/moko-resources/assets/5010169/c07b8cad-ae95-46c3-817a-d0ae081b6e81">

**Comment by sunildhiman90** [comment]
oh, i thought only gradle sync would be enough thats why i didnt tried generateMRcommonMain as it was mentioned either gradle sync or run this task to generate MR resource, 

Thanku, i will try generateMRcommonMain

**Comment by Alex009** [comment]
yes, you can use gradle sync or gradle task. you do gradle sync after resources dir changes or before?

**Comment by sunildhiman90** [comment]
I did sync after resources dir changes, But it didnt updated, ANd one more thing, now i tried using `./gradlew generateMRcommonMain`, But still its not producing resoruces, Am i still missing anything? 
But as you also checked that branch, it worked for you ,So what can be the issue from my side? any idea?

**Comment by Alex009** [comment]
maybe you expect something wrong? what exactly you expect from `./gradlew generateMRcommonMain` run?
also try to run `./gradlew generateMRcommonMain --info` and attach logs here

**Comment by sunildhiman90** [comment]
Hi, i have attached the logs file of ```./gradlew generateMRcommonMain --info```

[logs (1).docx](https://github.com/icerockdev/moko-resources/files/11715332/logs.1.docx)

**Comment by harsh2907** [comment]
Hey Alex, may i know the name of the theme you use in android studio

**Comment by malliaridis** [comment]
@sunildhiman90 can you try setting the JVM version to 11, clean cache and rebuild the project? **I had to set the Gradle JVM to 11** as well as the Android `targetCompatibilty` and `sourceCompatibilty` in the `android.compileOptions` to 11, then it worked. 

The Gradle JVM is the only thing that is not version-controlled and probably the reason why it worked for Alex.

@harsh2907 It's the [New UI](https://www.jetbrains.com/help/idea/new-ui.html) of the JetBrains IDEs. I am not sure if it is already available for Android Studio, but all the other IDEs of JetBrains have this UI now.

**Comment by tassilo-posegga** [comment]
Same issue after trying to rename package names after using the KMP Compose Template. Resources are not generated with the new package defined
```
multiplatformResources {
    multiplatformResourcesPackage = "foo.bar"
}
```

**Comment by Alex009** [comment]
@tassilo-posegga did you use 0.24.0 alpha 5 version?

**Comment by ExNDY** [comment]
"Closing this issue due to inactivity as there has been no response for a long time. If this is still an issue for you, please feel free to reopen it or create a new one with updated details."
